src/test/ui/consts/rfc-2203-const-array-repeat-exprs/migrate-fail2.rs

```diff
@@ -0,0 +1,18 @@
+// ignore-tidy-linelength
+// ignore-compare-mode-nll
+// compile-flags: -Z borrowck=migrate
+// build-fail
```

Comment:
this is necessary right now, because with `check-pass` the test would succeed. I'm currently looking into it.

Reply:
This is a general problem with `--emit=metadata`, we don't compute the `optimized_mir` for functions. It will be fixed when we address that.

Reply:
Is there an issue tracking this?

Reply:
I don't believe so and a quick search through our issues didn't show anything. I'll open something...